### ISSUE TITLE
Consider ordinal variables, such as the party id 7 point scale.

### ISSUE THREAD

**Issue by dbroockman** [acoppock/Green-Lab-SOP#4]

**Comment by acoppock** [comment]
Yes, thanks!  You've prompted us to think about default procedures for the various types of dependent variables that crop up in experiments.  

We were thinking that (in cases where the PAP doesn't specify how such variables are to be analyzed) we would advocate difference-in-means (or OLS) for all numeric variables. Categorical dependent variables are obviously more tricky -- people often turn them into binary variables and thereby avoid multinomial complications.  Ordinal is an intermediate case.  I guess if pid were on the left hand side (and the PAP didn't specify) I would advocate diff-in-means or OLS as a default.  But other ordinal variables (like your sliding scale of meeting prestige :) ) would be odd to analyze that way, since they really are more like categories than a scale.    

This is clearly a case were PAPs are hugely valuable -- the question is, what's the right default for when PAPs are silent on the issue?

**Comment by dbroockman** [comment]
I think the right default is OLS because usually we want point estimates that give us some sense of what is going on.

Note that factor analysis is almost always going to be making this decision implicitly if the mapping between the items and the latent variable is linear.

For the meeting thing the graph was better at communicating the nature of the effect because what the categories meant was so clear.
